### ISSUE TITLE
CSS in Javascript file breaks.

### ISSUE THREAD

**Issue by leanst-daniel** [vaadin/flow#9355]
Hello everyone!

We're building our Vaadin apps in a Docker container, which has Maven 3.6.3-jdk-8 as a base image. For Node we're installing 12.8.

Occasionally it happens that our build runs through perfectly fine but the actual UI looks like the CSS is entirely missing.
Means that all text and embedded pictures are there but no format or buttons.

A brief investigation has shown that some parts of the CSS definition in vaadin-bundle-****.js are missing. 

So far it's not reproducible and happens roughly 2 times out of 10 builds, but it can happen many more time after the occurance. It's not browser or client related. Also not related to the host that we're building on, since it happened on different instances already. 
For troubleshooting we've upgraded Vaadin, NodeJS, Maven and the Docker Base Image.

    - Vaadin / Flow version: 14.4.1
    - Java version: 8
    - OS version: openjdk8:jre8u252-b09-alpine

**Comment by taefi** [comment]
Hi @leanst-daniel, have you experienced the issue so far after upgrading Vaadin and the rest of the tools and images? If yes, would you please provide some information/instruction for us to reproduce it?

**Comment by leanst-daniel** [comment]
Hi @taefi , no we've experienced this issue throughout the last 3 latest versions.

What we've tried is to install NodeJS ourselfs before the Maven build starts, let Vaadin install NodeJS, different Maven versions and different underlying Docker images. We never found out how to reproduce the issue because it only happened roughly in 2 out of 10 builds.

It's not an issue for us anymore because we switched the build process from a "normal" server to a GitLab runner and didn't have any issues so far.
